Frontend/ui-driver/src/Components/StatsModel/View.purs

```diff
@@ -54,53 +54,86 @@ earningsView config push =
   , gravity CENTER
   , padding (Padding 0 5 0 5)
   , cornerRadius 9.0
-  ][ linearLayout
-        [ width $ V ((screenWidth unit) /2 - 16)
-        , height WRAP_CONTENT
-        , orientation HORIZONTAL
-        , gravity CENTER
-        ][ textView $ 
-            [ height config.countTextConfig.height
-            , text config.countTextConfig.text
-            , margin $ MarginRight 10
-            , gravity config.countTextConfig.gravity
-            , color config.countTextConfig.color
-            -- , weight config.countTextConfig.weight
-            ] <> FontStyle.tags TypoGraphy
-         , textView $
-            [ height config.textConfig.height
-            , text $ show $ config.totalRidesOfDay
-            , gravity config.textConfig.gravity
-            , width WRAP_CONTENT
-            , color config.textConfig.color
-            -- , weight config.textConfig.weight
-            ] <> FontStyle.h2 TypoGraphy
-         ]
+  ][  linearLayout
+      [ width WRAP_CONTENT
+      , height WRAP_CONTENT
+      , orientation VERTICAL
+      , gravity CENTER
+      , weight 1.0
+      ][ textView $ 
+         [ height config.countTextConfig.height
```

Comment:
Make a common textview for all the textviews that are being used in this layout.